content/08.transmission.md

```diff
@@ -1,24 +1,68 @@
+The incubation period, or time period between exposure and the onset of symptoms, has been estimated at five to eight days (4.91 and 7.54 in two different cities) [@doi:10.7554/eLife.57149; @doi:10.1164/rccm.202003-0524LE], and estimates suggest that viral shedding may begin long before the onset of symptoms (12.3 days with a 95% CI of 5.9 - 17.0) and peak around the onset of symptoms [@doi:10.1038/s41591-020-0869-5].
+As these trends became apparent, concerns arose due to the potential for individuals who did not or did not yet show symptoms to transmit the virus [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html].
+Recovered individuals may also be able to transmit the virus after their symptoms cease.
+Estimates of the communicable period based on twenty-four individuals who tested positive for SARS-CoV-2 prior to or without developing symptoms estimated that individuals may be contagious for one to twenty-one days, but they note that this estimate may be low [@doi:10.1007/s11427-020-1661-4].
+Initially, viral nucleic acids were reported to remain at observable levels in the respiratory specimens of recovering hospitalized COVID-19 patients for a median of 20 days and with a maximum observed duration through 37 days, when data collection for the study ceased [@doi:10/ggnxb3].
+As more estimates of the duration of viral shedding are released, they are beginning to converge around approximately three weeks from first positive PCR test and/or onset of symptoms (which, if present, are usually identified within three days of the initial PCR test).
+For example, viral shedding was observed for up to 28 days following symptom onset in a second study [@doi:10.1038/s41586-020-2196-x] and for one to 24 days from first positive PCR test with a median of 12 days [@doi:10.1001/jama.2020.3204].
+On the other hand, almost 70% of patients were reported to still have symptoms at the time that viral shedding ceased, although all symptoms reduced in prevalence between onset and cessation of viral shedding (CVS) [@10.1016/j.jcv.2020.104577].
+They also reported that the median time that elapsed between the onset of symptoms and CVS was 23 days and between first positive PCR test and CVS was 17 days [@10.1016/j.jcv.2020.104577]. 
+This study reported that symptom onset predated the first positive PCR test by an average of three days, however, which suggests that there may be some methodological differences between it and related studies.
+An analysis of residents of a nursing home with a known SARS-CoV-2 case found similar rates of SARS-CoV-2 positive individuals who were asymptomatic and did not (25.5) or did (23.1) later develop symptoms, those who had atypical symptoms (24.2), and those who had typical symptoms (24.8) [@doi:10.1056/NEJMoa2008457].
```

Comment:
The meaning of the brackets wasn't clear here, I went the the paper and it said it was the cycle thresholds. Is that required?

Reply:
I'm almost sure these are percentages... not sure what was wrong with me while writing this 😬  I'll double-check and add units!

Reply:
HaHa no worries :)